packages/app/src/containers/Canvas.tsx

```diff
@@ -249,6 +255,68 @@ export default memo(function Canvas() {
           event.preventDefault();
           break;
         }
+        case 'editPath': {
+          const layer: any = Selectors.getLayerAtPoint(
```

Comment:
`getLayerAtPoint` will affect path selection in a way we don't want, since the POINT_RADIUS extends past the layer's edge. This is probably what's making it harder to click

Reply:
replaced with `getSelectedLayers()`